src/audio/CMakeLists.txt

```diff
@@ -24,6 +17,21 @@ if(NOT BUILD_HOST)
 		component.c
 		buffer.c
 	)
+	if(CONFIG_COMP_VOLUME)
+		add_local_sources(sof
```

Comment:
How about moving to a subdir and stop maintaining the list at this components top level?

Reply:
@mmaka1 First of all host build is weird and it shouldn't look like this. Anyway as long as it has the same buildsystem as FW - it should be here, otherwise you'd have subdir that takes care of objects from parent dir which is against cmake's practices